src/hooks/usePathForNetwork.ts

```diff
@@ -11,12 +11,12 @@ export function usePathForNetwork({
   history,
 }: {
   pathName: string;
-  networkID: number;
+  networkID: NetworkId;
   history: History;
 }) {
   const handlePathForNetwork = () => {
     // do nothing if networkID is -1 since that's a default state
-    if (networkID === -1) return;
+    // if (networkID === -1) return;
```

Comment:
this may be a problem, please check

Reply:
yea with the current way networkID is initialized in redux we can have -1... I actually fixed that annoyance here, but I need someone to review https://github.com/OlympusDAO/olympus-frontend/pull/1005